tests/Tests.AzureAppConfiguration.Functions.Worker/RefreshMiddlewareTests.cs

```diff
@@ -1,14 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
+using System;
```

Comment:
About sorting usings, current order looks like alphabetical, and you place `System` directives first. Is there any specific benefit to change it? Or a convention we didn't follow? I'm asking because the change affects almost all files. Can we just keep it as is and make the corresponding linting rule to match the current sorting strategy?

Reply:
This is the default settings of .NET formatting options. 
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/dotnet-formatting-options#dotnet_sort_system_directives_first

> Can we just keep it as is and make the corresponding linting rule to match the current sorting strategy?

`dotnet_sort_system_directives_first = false` will just sort the all using directives in alphabetical order.

Reply:
Discussed with Jimmy, we will follow the visual studio behavior: 
![image](https://github.com/Azure/AppConfiguration-DotnetProvider/assets/141655842/ac694b07-1258-4eba-a29b-f5795caa10c7)
The `Sort Usings` function provided by visual studio has the same behavior as `dotnet_sort_system_directives_first = false` does.

Reply:
AFAIU, the behavior of "Sort Usings" is matching with your VS settings. And if you check "place System directives first" in settings, I assume it does put System on top. 

BTW, I'm ok with either style if there's no side-effects. And I also vote for the alphabetical order because:
a) the order of usings in our code is alphabetical now, 
b) VS default behavior is alphabetical out-of-box, as the goal is to unify the code styles instead of bringing obstacles for contributors.